cmd/kubeless/function/deploy.go

```diff
@@ -85,6 +85,11 @@ var deployCmd = &cobra.Command{
 			logrus.Fatal(err)
 		}
 
+		fromURL, err := cmd.Flags().GetString("from-url")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
 		file, err := cmd.Flags().GetString("from-file")
```

Comment:
we would need to throw an error if `--from-url` and `--from-file` are both set

Reply:
did you miss this comment? (not sure if it has been answered elsewhere). Actually, thinking about it, it would be better to support that the flag `--from-file` supports to specify an URL. That is more similar to `kubectl` that allows you to do `kubectl create -f https://...` we can parse the path and discover if it is an URL.